app/uk/gov/hmrc/personaldetailsvalidation/PersonalDetailsValidator.scala

```diff
@@ -70,7 +72,7 @@ class PersonalDetailsValidatorImpl @Inject() (
             case _ => attempts.map { attempts => personalDetailsValidationRetryRepository.recordAttempt(maybeCredId.get, attempts) }
           }
         }
-        personalDetailsValidationRepository.create(personalDetailsValidation)
+        repoControlService.insertPDVAndAssociationRecord(personalDetailsValidation, maybeCredId, hc)
```

Comment:
can we make sure an IT test covers this scenario. i.e hit the route, 
both db's inserted into
if association db not written to, submission still successful
if association db already has record in submission still succeeds and does not affect the user